samples/quickstart.js

```diff
@@ -0,0 +1,43 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+'use strict';
+
+// eslint-disable-next-line no-unused-vars
+async function main(projectId) {
+  // [START area120_tables_quickstart]
+  // Imports the Google Cloud client library
+
+  // eslint-disable-next-line node/no-extraneous-require, node/no-missing-require
```

Comment:
this shouldn't be necessary as long as the linking logic is correct in `samples/package.json` and you you've installed and linked everything before you run `lint`. This `prelint` task helps:

"prelint": "cd samples; npm link ../; npm i"